### PR TITLE
Updating startup to explicitly look for debugger request

### PR DESCRIPTION
**Pull request by fabiocav**

### CODE REVIEW

**Comment by SeanFeldman** [comment]
@fabiocav will this allow more time to be spent in the debugger w/o timing out?

**Comment by fabiocav** [comment]
No, this is primarily to enable profiling flows

File: src/DotNetWorker.Core/StartupHook.cs [hunk 1]
```diff
@@ -48,9 +42,19 @@ static bool WaitOnDebugger(int cycle)
...
-        for (int i = 0; WaitOnDebugger(i); i++)
+        if (string.Equals(jsonOutputEnabled, bool.TrueString, StringComparison.OrdinalIgnoreCase))
+        {
+            Console.WriteLine($"azfuncjsonlog:{{ \"name\":\"dotnet-worker-startup\", \"workerProcessId\" : { Environment.ProcessId } }}");
+        }
+
+        if (string.Equals(debuggerWaitEnabled, bool.TrueString, StringComparison.OrdinalIgnoreCase))
         {
-            Thread.Sleep(SleepTime);
+            Console.WriteLine($"Azure Functions .NET Worker (PID: { Environment.ProcessId }) initialized in debug mode. Waiting for debugger to attach...nblswdivnsldvnsldiv");
```
Comment (ankitkumarr) [inline src/DotNetWorker.Core/StartupHook.cs:52]: `attach...nblswdivnsldvnsldiv` I am guessing you didn't mean to type that?

Comment (fabiocav) [reply src/DotNetWorker.Core/StartupHook.cs:52]: It was just a test to make sure you were really paying attention when reviewing. 😆 

Sorry, fixing :)

**Review by ankitkumarr — APPROVED** [review]

File: src/DotNetWorker.Core/StartupHook.cs (hunk 1 above)
Comment (SeanFeldman) [reply src/DotNetWorker.Core/StartupHook.cs]: Someone's sleeping on the job 😂

**Pull request merged**